events-api-ht-3/utils.js

```diff
@@ -0,0 +1,114 @@
+const fs = require('fs');
+const Transform = require("stream").Transform;
+
+const csvFile = "events.csv";
+const lineBreak = '\n';
+
+
+async function getEventsByLocation(location) {
+  const fileContent = await readFile();
```

Comment:
One possible improvement could be to NOT read all the data into the memory but filter needed data while reading (via streaming).